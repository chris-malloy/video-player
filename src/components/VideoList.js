import React from 'react';
import VideoListItem from './VideoListItem'

export default function VideoList(props){
    return (
        <ul className="col m4">
        </ul>
    )
}