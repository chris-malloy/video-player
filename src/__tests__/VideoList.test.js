import React from 'react';
import VideoList from '../components/VideoList';
import renderer from 'react-test-renderer';

describe('Component: VideoList', () => {

    test('VideoList renders correctly', () => {
        const component = renderer.create(
            <VideoList>VideoList</VideoList>
        )
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
});