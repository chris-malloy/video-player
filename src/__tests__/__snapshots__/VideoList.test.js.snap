// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: VideoList VideoList renders correctly 1`] = `
<ul
  className="col m4"
/>
`;
